component/prometheus/operator/podmonitors/podmonitor_test.go

```diff
@@ -0,0 +1,231 @@
+package podmonitors
```

Comment:
shall we rename this file to `config_gen_pod_monitor_test.go`? (And config_gen_podMonitor.go should probably be named `config_gen_pod_monitor.go`?)